src/main/java/org/jabref/model/entry/BibEntry.java

```diff
@@ -622,6 +622,15 @@ public String getAuthorTitleYear(int maxCharacters) {
     }
 
     /**
+     * Returns the title of the given BibTeX entry as an optional.
+     *
+     * @return the title of a BibTeX entry or null if no title was found.
+     */
+    public Optional<String> getTitle() {
+        return getField(FieldName.TITLE);
```

Comment:
Not sure what @JabRef/developers think about returning an optional there, but I think instead the error handling (absence of title) should be done already in this getter.
Thus I would vote for returning an empty string of the optional is absent.